nodejs/packages/layer/package.json

```diff
@@ -23,32 +23,32 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/api": "^1.2.0",
     "@opentelemetry/exporter-trace-otlp-proto": "^0.31.0",
     "@opentelemetry/exporter-metrics-otlp-proto": "^0.31.0",
     "@opentelemetry/instrumentation": "^0.31.0",
     "@opentelemetry/instrumentation-aws-lambda": "^0.32.0",
-    "@opentelemetry/instrumentation-aws-sdk": "^0.8.0",
+    "@opentelemetry/instrumentation-aws-sdk": "^0.8.1",
     "@opentelemetry/instrumentation-dns": "^0.29.0",
     "@opentelemetry/instrumentation-express": "^0.30.0",
     "@opentelemetry/instrumentation-graphql": "^0.29.0",
     "@opentelemetry/instrumentation-grpc": "^0.31.0",
```

Comment:
"^0.32.0" is available for us to update 🙂

Reply:
updated :)